inductiva/fluids/scenarios/fluid_tank/fluid_tank.py

```diff
@@ -191,7 +192,42 @@ def simulate(
             output_time_step=self.output_time_step,
             netcdf_data_dir=os.path.join(output_path, "netcdf"))
 
-        return output_path
+        return FluidTankOutput(output_path)
+
+    def simulate_async(
+        self,
+        simulator: Simulator = SPlisHSPlasH(),
+        resource_pool_id: Optional[UUID] = None,
+        device: Literal["cpu", "gpu"] = "cpu",
+        simulation_time: float = 5,
+        resolution: Literal["low", "medium", "high"] = "low",
+        output_time_step: float = 0.1,
+        particle_sorting: bool = False,
+    ):
+        """Simulates the scenario.
+
+        Args:
+            simulator: Simulator to use.
+            device: Device in which to run the simulation.
+            simulation_time: Simulation time, in seconds.
+            output_time_step: Time step between output files, in seconds.
+            resolution: Resolution of the simulation. Controls the particle
+                radius and time step.
+            particle_sorting: Whether to use particle sorting.
+        """
+
+        self.simulation_time = simulation_time
+        self.particle_radius = ParticleRadius[resolution.upper()].value
+        self.time_step = TimeStep[resolution.upper()].value
+        self.output_time_step = output_time_step
+        self.particle_sorting = particle_sorting
+
```

Comment:
I think here is missing the conversion to netcdf data. Otherwise, running async want allows to obtain any vizs after downloading the outputs.